build/release.mk

```diff
@@ -31,11 +31,23 @@ ifeq ($(GENERATE_PACKAGES), true)
 	# Pre-release/release actually builds and uploads images
 	# goreleaser will compile binaries, generate manifests, and push multi-arch docker images
 	# TAG_SUFFIX should be set as "-pre" during prereleases
-	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser.yml --skip-validate --rm-dist
+	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser.yml --skip=validate --clean
 else
 	@echo "===> $(INTEGRATION) === [release/build] build compiling all binaries"
 	# release/build with PRERELEASE unset is actually called only from push/pr pipeline to check everything builds correctly
-	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip-validate --snapshot --rm-dist
+	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip=validate --snapshot --clean
+endif
+
+.PHONY : release/build-fips
+release/build-fips: release/deps release/clean
+ifeq ($(GENERATE_PACKAGES), true)
+	@echo "===> $(INTEGRATION) === [release/build] PRERELEASE/RELEASE compiling fips binaries, creating packages, archives"
+	# TAG_SUFFIX should be set as "-pre" during prereleases
+	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser-fips.yml --skip=validate --clean
+else
```

Comment:
this if-else condition looks useless, as far as I see GENERATE_PACKAGES is always true, also in the existing code! I think we should remove the dead code

Reply:
It is used for push/pr workflow where, by default it is `false` and only binaries are built.

Reply:
are we calling make release?